kubejs/data/ptd/advancements/raw_white.json

```diff
@@ -2,17 +2,20 @@
   "parent": "ptd:white_powder",
   "criteria": {
         "0": {
-            "trigger": "minecraft:impossible",
-            "__comment": "triggered by lychee executing command",
-            "conditions": { }
+            "trigger": "minecraft:inventory_changed",
```

Comment:
Why has this changed to inventory_changed? Previously it was achievable by using white dye on white powder using a create deployer or by hand. Now they have to pick it up too. It won't be able to be achieved if left on a belt

Reply:
Revert this change. Agreed.

Reply:
Additionally, maybe something similar to this should be done for the sturdy device advancement.